CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 - PRODUCTION : Creation of a new object MPS Charge
 - MACHINE : Tool management
 - PRODUCTION : created buttons in product to create new bill of material and production process.
-
+- PURCHASEORDERLINE : Replaced salMinPrice per maxPurchasePrice
```

Comment:
Changelog is not meant only for technical people. Avoid using technical names of field and make a more "verbal" description of the change.
(Also grammar nazi : use "replaced by" instead of "replaced per").